scripts/ci/install-minikube.sh

```diff
@@ -1,19 +1,18 @@
 #!/bin/bash
 
-set -v
-
 OS=linux
 ARCH=amd64
 TARGET_DIR="/usr/bin"
 
-MINIKUBE_VERSION="v0.24.1"
+
+MINIKUBE_VERSION="v0.25.2"
 MINIKUBE_URL="https://github.com/kubernetes/minikube/releases/download/$MINIKUBE_VERSION/minikube-$OS-$ARCH"
 
 KUBECTL_VERSION="v1.9.0"
 KUBECTL_URL="https://storage.googleapis.com/kubernetes-release/release/$KUBECTL_VERSION/bin/$OS/$ARCH/kubectl"
 
-[ -z "$MINIKUBE_FLAGS" ] && MINIKUBE_FLAGS="--extra-config=kubelet.CgroupDriver=systemd"
```

Comment:
please keep the service configuration variables on to of the files

Reply:
ok. moved to top